addon_service/abstract/configured_addon/models.py

```diff
@@ -49,7 +49,8 @@ def display_name(self):
 
     @display_name.setter
     def display_name(self, value: str):
-        self._display_name = value
+        if value:
+            self._display_name = value
 
```

Comment:
maybe use this approach will be better + typing fix

```
def display_name(self, value: str) -> str | None:
    return  self._display_name if value else None
```

Reply:
this is a setter, therefore it returns nothing